azerothcore/azerothcore-wotlk
# fix(DB/SmartScript) Dreghood Brute dont become friendly after the death of the taskmaster

## Changes Proposed:
-  Creates the event by killing the Illidari Taskmaster
```
Smart_Scripts
Script 0  /*on Illidari Taskmaster death update dreghood brute template to Fleeing Dreghood Warrior*/
Script 1 /*on Illidari Taskmaster death update dreghood geomancer template to Fleeing Dreghood Warrior*/
Script 2 /*on Illidari Taskmaster death set Fleeing Dreghood Warrior faction to 15 (creature, neutral)*/
Script 3 /*on Illidari Taskmaster death set emote to 470 (STAND_STATE_NONE)*/
Script 4 /*on Illidari Taskmaster death, Fleeing Dreghood Warrior stop attacking the player*/
Script 5 /*on Illidari Taskmaster death, Fleeing Dreghood Warrior says runaway text*/
Script 6 /*on Illidari Taskmaster death, Fleeing Dreghood Warrior start random movement*/
Script 7 /*on Illidari Taskmaster death, Fleeing Dreghood Warrior Despawn In 10000 ms*/
```

## Issues Addressed:

- https://github.com/azerothcore/azerothcore-wotlk/issues/13589
- https://github.com/chromiecraft/chromiecraft/issues/4331

## SOURCE:

https://www.youtube.com/watch?v=hiGokJ9K438
https://www.youtube.com/watch?v=YXCqgLo5ihk
https://www.youtube.com/watch?v=lUH81CeTXOs

## Tests Performed:

- 
![WoWScrnShot_110422_161205](https://user-images.githubusercontent.com/111697701/200077051-6ff60a48-16ac-45e2-a179-abf98f9bd11a.jpg)

- 
![WoWScrnShot_110422_161209](https://user-images.githubusercontent.com/111697701/200077070-07dc18e9-a553-451c-bb4c-cb9a947d2088.jpg)



## How to Test the Changes:


1. Insert the sql in your database and then go to the zone, kill the taskmaster and then see what happens

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
